src/main/java/com/github/manolo8/darkbot/gui/MainGui.java

```diff
@@ -89,4 +123,17 @@ public void tick() {
         mapDrawer.repaint();
     }
 
+    private static boolean isOutsideScreen(Config.BotSettings.Window window) {
+        GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        for (GraphicsDevice device : ge.getScreenDevices()) {
+            Rectangle deviceBounds = device.getDefaultConfiguration().getBounds();
+            if (window.height > deviceBounds.height || window.width > deviceBounds.width)
+                return true;
+            if (!device.getDefaultConfiguration().getBounds()
+                    .intersects(new Rectangle(window.x, window.y, window.width, window.height)))
+                return true;
```

Comment:
for multiple monitor setups this will always be true, as at least one of the screens will say that it does NOT intersect the bot.

Also, we do not want to check if it *intersects* (that means, at least one tiny corner happens to be in the screen) we want to check that it is fully contained in a screen (or screens). I'd check that both top-left and bottom-right corners are inside one of the screens